cmd/viam-agent/main.go

```diff
@@ -164,6 +164,16 @@ func main() {
 		}
 	}
 
+	netAppender, err := manager.NetAppender(ctx)
```

Comment:
This likely needs to be after the "fast start" section just below. Fast start is supposed to work entirely offline and as quickly as possible (specifically for tennibot) and this leads to dialing (and timeouts) vastly slowing that offline behavior I believe.

Might be best for this to live within the dial() function? E.g. it'll set up the network appender whenever anything first dials, regardless of source.

Reply:
felt initially weird to put it in dial, then realized it *has* to be dial because we don't know when we will come online

done in ec27bc3

should I do anything to protect against a second netappender getting added on reconnect? I think this doesn't happen now, but could see it accidentally happening if we refactor in the future

(for example, do `SetNetAppender` instead of `AddNetAppender`)